feature/amazon/src/test/java/com/revenuecat/purchases/amazon/AmazonOfferingsTest.kt

```diff
@@ -83,7 +130,8 @@ class AmazonOfferingsTest {
         )
         val annualPackageToTest = offeringsParser.createPackage(annualPackageJSON, productsMap, "offering")
         Assertions.assertThat(annualPackageToTest).isNotNull
-        Assertions.assertThat(annualPackageToTest!!.product).isEqualTo(productsMap[storeProductAnnual.id]?.get(0))
+        Assertions.assertThat(annualPackageToTest!!.product).usingRecursiveComparison()
```

Comment:
`usingRecursiveComparison` checks field-by-field, we need this now since the packages are copied

Reply:
oh, i think it's because our dummies return an implementation of StoreProduct, not AmazonStoreProduct.